mcu_cfg.hjson

```diff
@@ -116,7 +116,7 @@
         pdm2pcm: {
             offset:  0x00060000,
             length:  0x00010000,
-            is_included: "no",
+            is_included: "yes",
```

Comment:
Please revert any RTL change

Reply:
Sorry about that, it's a bit weird because I didn't touch anything there. Anyway, I reverted everything to the default state.

Reply:
LGTM thanks - I leave to @JoseCalero the final review and merge as he is the SW leader